docs/pages/powermanager_tests.md

```diff
@@ -0,0 +1,139 @@
+- Wifi wake up
+- `IR` wake up
+- Power key wake up
+- Timer wake up
+- `CEC` wake up
+- `LAN` wake up
+
+## Set And Get Power States
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Set and Get Power States](#set-and-get-power-states)|Set various power states and retrieve it for verification |PLAT_API_SetPowerState, PLAT_API_GetPowerState|Y|`NA`|`NA`|`NA`|
+
+### Test Startup Requirement - Set And Get Power States
+
+`NA`
+
+### Emulator Requirements - Set And Get Power States
+
+`NA`
+
+### Control Plane Requirements - Set And Get Power States
+
+`NA`
+
+## Set and Get Status of Wakeup Sources
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Set and Get Status of Wakeup Sources](#set-and-get-status-of-wakeup-sources)|Set status of various wakeup sources and retrieves status for verification |PLAT_API_SetWakeupSrc, PLAT_API_GetWakeupSrc|Y|`NA`|`NA`|`NA`|
+
+### Test Startup Requirement - Set and Get Status of Wakeup Sources
+
+`NA`
+
+### Emulator Requirements - Set and Get Status of Wakeup Sources
+
+[Emulator Requirements](#emulator-requirements)
+
+### Control Plane Requirements - Set and Get Status of Wakeup Sources
+
+`NA`
+
+## Testing Wakeup Source
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Testing Wakeup Source](#testing-wakeup-source)|Power manager should accept the supported wakeup sources and device should wakeup from standby/sleep modes using wakeup sources set by caller|PLAT_API_SetWakeupSrc, PLAT_DS_SetDeepSleep|`NA`|Y|Control plane to trigger the wakeup source|
+
+### Test Startup Requirement - Testing Wakeup Source
+
+`NA`
+
+### Emulator Requirements - Testing Wakeup Source
+
+[Emulator Requirements](#emulator-requirements)
+
+### Control Plane Requirements - Testing Wakeup Source
+
+Control plane to trigger the wakeup source supported by platform
+
+## Test Reset Functionality
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Test Reset Functionality](#test-reset-functionality)|Power manager should reboot the device|PLAT_Reset|`NA`|Y|`NA`||
+
+### Test Startup Requirement - Test Reset Functionality
+
+`NA`
+
+### Emulator Requirements - Test Reset Functionality
+
+`NA`
+
+### Control Plane Requirements - Test Reset Functionality
+
+- Discuss options to verify that the device as rebooted successfuly.
+    - Monitor power usage via control panel
+    - Verify that the device has rebooted via connectivity with another device
+    - Verify that the reboot as occurred by reconnecting to the device and calling linux `uptime` command
```

Comment:
Also, required to mention that "Raft framework enables the test to resume from the point where it was interrupted due to a reboot."

Reply:
Added it in as requested